README.md

```diff
@@ -1,4 +1,7 @@
-# QuDynamics.jl
+QuDynamics.jl
+============
+
+[![Build Status](https://travis-ci.org/JuliaQuantum/QuDynamics.jl.png?branch=master)](https://travis-ci.org/JuliaQuantum/QuDynamics.jl)
```

Comment:
I was going through two other packages and they seem to have it, may be its specific to master branch, I might be wrong but was in that thinking.